templates/cases/case_read.haml

```diff
@@ -151,9 +151,31 @@
                   [[ key ]]
                 : [[ val ]]
 
+        - for pod in pods
+            %div{
+              ng-controller:"={ pod.pod_type.controller }",
+              ng-init:'init && init(={ pod.config.index }, ={ case_id }, ={ pod.config_json })',
+              "={ pod.pod_type.directive }":"true"
+            }
```

Comment:
The existing directive cpContact is used as an element <cp-contact /> rather than attribute <div cp-contact.. >.  I don't mind which approach we use but would like to be consistent.